src/Fields/DateRangePicker.php

```diff
@@ -49,17 +49,19 @@ class DateRangePicker extends Field implements HasAffixActions
     protected bool | Closure $useRangeLabels = false;
     protected bool|Closure $disableRange = false;
     protected bool | Closure $disableCustomRange = false;
+    protected bool | Closure $disableInputClearing = false;
     protected string $separator = ' - ';
 
-    public static function make(string $name) : static
+    public function withInputClearingAction() : static
```

Comment:
This change would not default the filter with clearing, please leave the make as it is and add the function to disable the clearning by overriding the suffixAction without 
```php
->action(fn() => $this->clear()))
```

Reply:
As well, the api would be disable as implemented before